mkdocs.yml

```diff
@@ -137,6 +137,7 @@ nav:
       - Regions in OpenStack: openstack-cloud-design-regions.md
       - Availability Zones in OpenStack: openstack-cloud-design-az.md
       - Host Aggregates in OpenStack: openstack-cloud-design-ha.md
+      - Rackspace Infrastructure Deployment: rackspace-infrastructure-deployment.md
```

Comment:
Rename rackspace to a general reference implementation/infrastructure to make it more agnostic